app/assets/javascripts/publishedToggle.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function() {
+  $("#auction_purchase_card").on("change", function(event) {
+    c2_status = $("#auction_c2_status").val()
+    published_select_element = $("#auction_published")
+
+    published_option = "<option value='published'>published</option>"
+    unpublished_option = "<option value='unpublished'>unpublished</option>"
+
+    if(this.value === "default" && c2_status !== "approved") {
+      published_select_element.empty().append(unpublished_option)
+    } else {
+      published_select_element.empty().append(published_option, unpublished_option)
```

Comment:
just tested this locally -- can we re-order these so that the default is still `unpublished` ? super small detail, but in general we do not publish auctions on create so it's nice to default to what we'd normally do